### PR TITLE
#1393 README show how to subscribe/unsubscribe

### PR DESCRIPTION
**Pull request by paulvi**
show how subscribe/unsubscribe to google groups by email

#1393 , close #1402

Signed-off-by: Paul Verest <paul.verest@live.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,8 @@ bnd is continuously built and released on [CloudBees](https://bndtools.ci.cloudb
 
 ## Feedback
 Feedback is always welcome, for general discussions use the <bndtools-users@googlegroups.com> list. We also have the <bndtools-dev@googlegroups.com> list for discussions on the development of bnd.
+One can subscribe to the bndtools-users mail list by sending an email to
+by sending an email to `bndtools-users+subscribe@googlegroups.com` and unsubscribe by sending an email to `bndtools-users+unsubscribe@googlegroups.com`.
```
Comment (bjhargrave) [inline README.md]: It says "by sending an email to" twice.

**Comment by bjhargrave** [comment]
Also, don't create a new PR when you fix your commit. Just amend the commit and push --force. That will update the PR to your new commit.

**Comment by paulvi** [comment]
I edit in browser. Double phrase is removed.

**Comment by bjhargrave** [comment]
Please squash the commits into a single commit and make sure your commit message includes a `Signed-off-by` as required by CONTRIBUTING. Then push force your updated commit. Thanks.

**Comment by paulvi** [comment]
@bjhargrave That is 2 lines of your hints, that I just wanted to put into user facing README.

While I understand that every contribution should follow rules, there's no convenient way to do it via browser.

**Pull request closed without merging**